contrib/spec/podman.spec.in

```diff
@@ -394,11 +394,11 @@ install -dp %{buildroot}%{_unitdir}
 
 %if %{with varlink}
 %{__make} DESTDIR=%{buildroot} install.python
-%endif # varlink
 
 # install libpod.conf
 install -dp %{buildroot}%{_datadir}/containers
 install -p -m 644 %{repo}.conf %{buildroot}%{_datadir}/containers
+%endif #varlink
```

Comment:
Is libpod.conf only needed to support varlink?  Not sure why this was moved.

Reply:
@jwhonce im going to fiddle with the spec anyway, ill peek at it